win32/openssl_engine_example/openssl_engine_example.vcxproj

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Since the example does not seem to be specific to Windows it needs to be added to Makefile and CMakeFile.txt too and buildable on linux/osx.

Reply:
done